### ISSUE TITLE
Development strategy suggestion: Please change your development priority strategy - equal treatment of usability/customization features and crypto features development

### ISSUE THREAD

**Issue by Sa-Ja-Di** [brave/brave-browser#22379]
It would be nice to see the Brave browser team not only giving their development focus fully towards the Crypto features inside the browser but instead usability/customization feature requests should be treated the same way importance wise like the team does with the Crypto features or at least give usability/customization feature requests/improvement requests a little bit of a higher priority too with implementing/optimizing.

Not everyone cares for Crypto features but people still use Brave for various reasoning and right now all the ones who are not after Crypto features are neglected quite a lot. 

Description :
--------------------
Brave has a seriously non-equal development mentality, usability feature wishes or customization feature wishes are getting ignored or delayed without plan to actually implement them and the only priority are crypto related features

Actual result:
--------------------
Brave team only prioritizes the Crypto features of the browser 

Expected result:
--------------------
Give Non-Crypto features a higher development priority too and stop neglecting your Non-Crypto users please

**Comment by Arandhras** [comment]
Hi Sajadi, I fully agree with your statement. I believe that Brave is currently the best browser out there but I would love to see some more improvement outside of the Crypto realm, and beyond what Chromium offers already out of the box. Especially I would like to see some more UI customization options, like positioning of the tabs, different (editable) colour schemes and editable button positions.

In other words, try to take up what browsers like Firefox decided to give up on because of "simplicity".

This should all be optional: if the user doesn't care about these features, it should be like as if the editability of the UI was never there.

**Comment by srirambv** [comment]
cc: @rebron @bsclifton

**Comment by bsclifton** [comment]
@Sa-Ja-Di @Arandhras thanks for the feedback 😄 The wallet features are optional and aren't part of the experience unless you want them to be. I can empathize with how it seems these features seem to be prioritized above other work.

For me personally at Brave, I get to oversee and help with general development for the Android, iOS, and Desktop (including Brave Rewards). There are separate teams for privacy features, Chromium upgrades, Brave Ads, Brave Wallet, Search, and more. There's always overlap where we help each other (reviews, working on common code, sharing best practices, etc) but we each have our own project boards and deliverables.

Most recently on Android, we've launched both Brave News and our VPN product. These both involved standing up a lot of code, finding and fixing bugs, and getting feedback as folks try it out. As we start to free up, we have some great projects lined up for upgrading the new tab page (more favorites, customization), adding a tab search, implementing stack view for tabs, and more (Playlist feature similar to iOS in the works).

On Desktop, we've been adding more features to Brave News and helping with the shared codebase used in Android also. We've got an updated shields UI which is all using native code (rather than extension APIs). We've added a Sidebar and are working through the complexities of merging this with Chrome's side-panel (live in Chromium 102). We're getting the Adblock settings page (brave://adblock) moved into our settings page which will help regional filters become more visible (and subscribing/editing rules easier). VPN is something I'm working hard on with folks too - it's already working nicely on Beta and Nightly using our development environment but I'm helping make sure we have a good cross-platform experience ready before it launches.  There were some big building blocks we had to stand up to make this happen (you can search GitHub for SKU SDK). Just recently, we did have a few folks join us on Desktop so I think you'll start to see development speed up a bit 😄 I know for sure there are more New Tab Page customization features coming and we'll be looking at Playlist and more (vertical tabs is something we want to look at)

I'd definitely recommend using Nightly if you're not already on it to get some of the features earlier. Thanks again for sharing your feedback with us. If you haven't already found/upvoted/left comments on GitHub issues that you want to see prioritized, please do that. It definitely helps us
